docs/apidesc.md

```diff
@@ -66,8 +66,8 @@ one exception is NICs, where the label is unique only on a per-node basis.
     project_connect_node        <project_label> <node_label>
     project_detach_node         <project_label> <node_label>
  
-    node_connect_network        <node_label> <nic_label> <network_label>
-    node_detach_network         <node_label> <nic_label>
+    node_connect_network        <node_label> <nic_label> <network_label> [ <channel> ]
+    node_detach_network         <node_label> <nic_label> <network_label>
```

Comment:
Making it optional could help make it backwards-compatible.  Some ways of doing this are:  remove the native VLAN; remove /everything/; remove the native VLAN if it's the only one [as @henn suggested].  I prefer the first---it's most parallel with how we're doing node_connect_network.